capellambse_context_diagrams/collectors/default.py

```diff
@@ -83,21 +76,22 @@ def _make_box_and_update_globals(
         made_boxes[obj.uuid] = box
         return box
 
-    def _make_owner_box(current: t.Any) -> t.Any:
-        if not (parent_box := global_boxes.get(current.owner.uuid)):
+    def _make_owner_box(obj: t.Any) -> t.Any:
```

Comment:
The collector function is very long. Too long if you ask me, I can't even remember what I read when I need to scroll down. We should move both functions `_make_box_and_update_globals` and `_make_owner_box` outside of the function. We can still leave the `_` because they should not be part of the API.